app/src/shared/components/ModalRoot/index.jsx

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import React, { type Node } from 'react'
+import Context from '$shared/contexts/Modal'
+
+type Props = {
+    children: Node,
+}
+
+class ModalRoot extends React.Component<Props> {
+    componentDidMount() {
+        this.forceUpdate()
```

Comment:
Why is the `forceUpdate` needed?

Reply:
Well, I do want to push `root` element down the tree (via context) but also make sure it's always a div – see the `render` method. It's not perfect, I know (who needs another rerender, right?). Wondering if there's another way…